docs/docs/interactions.md

```diff
@@ -4,31 +4,31 @@ title: Cross handler interactions
 sidebar_label: Cross handler interactions
 ---
 
-Gesture handlers can "communicate" with each other in order to allow for defining complex gesture interactions and to control how they [activate](state.md#active) in certain scenarios.
+Gesture handlers can "communicate" with each other to support complex gestures and control how they *[activate](state.md#active)* in certain scenarios.
 
-There are two means of such a control at the moment described in the sections below.
-In each of the cases it is necessary to provide a reference of one handler as a property to the other.
+There are two means of achieving that described in the sections below.
+In each case, it is necessary to provide a reference of one handler as a property to the other.
 Gesture handler relies on ref objects created using [`React.createRef()`](https://reactjs.org/docs/refs-and-the-dom.html) and introduced in [React 16.3](https://reactjs.org/blog/2018/03/29/react-v-16-3.html#createref-api).
 
 ## Simultaneous recognition
 
-By default at a given time only one gesture handler is allowed to be in an [active](state.md#active) state.
-So when a gesture handler recognizes a gesture, it [cancels](state.md#cancelled) all the other handlers in [began](state.md#began) and prevents any new handlers from receiving a stream of touch events for as long as it is [active](state.md#active).
+By default, only one gesture handler is allowed to be in the [`ACTIVE`](state.md#active) state.
+So when a gesture handler recognizes a gesture it [cancels](state.md#cancelled) all other handlers in the [`BEGAN`](state.md#began) state *and* prevents any new handlers from receiving a stream of touch events as long as it remains [`ACTIVE`](state.md#active).
```

Comment:
```suggestion
So when a gesture handler recognizes a gesture it [cancels](state.md#cancelled) all other handlers in the [`BEGAN`](state.md#began) state and prevents any new handlers from receiving a stream of touch events as long as it remains [`ACTIVE`](state.md#active).
```

The emphasis we talked about.

Reply:
Yea this is just a preference matter in my opinion